Solutions/Menes.Abstractions/Menes/OpenApiServiceExtensions.cs

```diff
@@ -230,7 +230,7 @@ private static string GetContentType<T>(T instance, string? contentType)
 
         private static string? GetContentTypeOrNull<T>(T instance)
         {
-            return ContentFactory.TryGetContentType(instance, out string contentType) ? contentType : null;
+            return ContentFactory.TryGetContentType(instance!, out string? contentType) ? contentType : null;
```

Comment:
The null suppression implies you are certain `instance` won't be null. It might be better to express that as a constraint:

```cs
static string? GetContentTypeOrNull<T>(T instance)
    where T: notnull
```

Then you should be able to remove the `!`, and more importantly, the compiler should then tell you if someone tries to call this with something that might in fact be null.